### ISSUE TITLE
Accessibility Request from Disability Benefits team: focus element when transitioning between multiple date range screens

### ISSUE THREAD

**Issue by mengA6** [department-of-veterans-affairs/va.gov-team#78730]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: Disability Benefits team 1
- **OCTO product owner:** @emilytheis 
- **OCTO design lead:** @shannonkford 
- **Product name:** 526EZ Disability Benefits application - toxic exposure screens
- **Product manager:**  @RakshindaAslam 
- **Slack channel:** [#benefits-disability-design](https://dsva.slack.com/archives/C053UDWMH7U)
- **Dedicated content writer on your team (if you have one):** `none`
- **Dedicated a11y specialist on your team (if you have one):** `none`
- Earlier CAIA intake ticket: [[CAIA Intake] Benefits Disability Experience 526 EZ: Toxic Exposure Screens#71168](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/71168)
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

Adding toxic exposure questions to existing 526EZ Disability Benefits Application. We have reviewed this with CAIA and governance a few times and have gotten great input. A new question just came up.

When transitioning between screens within a sequence of screens that asks for a date range when the user served in a location or was exposed to a hazard as in the below example:

![TE multiple date screens](https://github.com/department-of-veterans-affairs/va.gov-team/assets/151068099/ebe3a79a-cb0c-43bc-9fbd-73835c90a9b9)

As this is implemented, the transition between screens is very subtle. It may not be clear to the user that they have transitioned from a screen asking for a date range they served in **Afghanistan** to a very similar screen asking for a date range when they served in **Iraq**, for example. 

We thought of addressing this by setting the focus to the heading `1 of 3: Afghanistan` or `2 of 3: Iraq` when the new screen loads. However, @christinec-fftc [discovered that](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/73848#issuecomment-1998089010) the initial focus is on the h2 `Step 2 of 5: Conditions`, which will be the same text for all of these screens. That seems to be the default initial focus element for the design system. If we changed the initial focus element, we would probably need to do that throughout the 526EZ form, which would be a significant effort in terms of design and implementation, so we want to get some feedback about the approach first.

What is your recommendation for how to make it clear, particularly for screen reader users that they have transitioned between these date range screens? 
 
## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

Ideally would get guidance by March 25, but it's also okay if it comes later 

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [ ] No
- [x] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

`Currently in implementation`

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [x] Design intent 
- [x] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** `60140`

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Figma screens](https://www.figma.com/file/dBKlB23Hs2oa53euXcXmmK/526-toxic-exposure?type=design&node-id=1236-54987&mode=design&t=OKCOIzVktAScYeAp-11)
- [VFS team epic](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/64536)

## Next steps
- [x] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [x] Review the remaining next steps in the **VFS team tasks** section here.

## Tasks

<br>

```[tasklist]
### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).
```

```[tasklist]
### CAIA operations
- [ ] Initial Review - @coforma-terry and @strelichl 
```

```[tasklist]
### CAIA a11y tasks
```

```[tasklist]
### CAIA content tasks
```

```[tasklist]
### CAIA IA tasks
```

**Comment by mengA6** [comment]
cc @Kate-allbee

**Comment by mengA6** [comment]
[Slack thread](https://dsva.slack.com/archives/C01K37HRUAH/p1710876495877119)

**Comment by eli-oat** [comment]
@mengA6 are ya'll using the [list and loop](https://design.va.gov/patterns/ask-users-for/multiple-responses) pattern for the multiple date inputs?

**Comment by mengA6** [comment]
@eli-oat kind of, but we are extending it since the existing list and loop pattern doesn't do all the things we are trying to do with it. we have this ticket to contribute our variation back to the design system:  [Experimental Design [enhancement to Multiple Responses - multi-page, from checkboxes] #2562 ](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2562)

**Comment by eli-oat** [comment]
Thanks so much @mengA6! Times like this are when I wish for dynamic steps...but that introduces a whole level of complication and headache. 

Is it possible to condense headings? `Service after August 2, 1990 in Iraq`?

If not, no worries -- there are some other options, but I'm hoping to understand what wiggle room we have here.

**Comment by mengA6** [comment]
Thanks for looking into it quickly @eli-oat . 

Possible, but that goes against a recommendation from [our IA feedback](https://app.zenhub.com/workspaces/disability-experience-63dbdb0a401c4400119d3a44/issues/gh/department-of-veterans-affairs/va.gov-team/75362)

> Bonus (current flow doesn't do this): For each item, indicate the current position in the list (something like Post-9/11 service location 1 of 3: Lebanon)

**Comment by eli-oat** [comment]
Wicked, that is good to know! Thank you @mengA6 

A few other things to kick around

## Option 1

At the very start of this flow, perhaps you could re-frame it: 

- "Did you serve in any of these location? If so, we'll ask you more questions on the following screens." 
- Then, follow up as an `h3` on the next page that replaces "Service after..." to the name of where they served, e.g. "Afghanistan." 

## Option 2

Also of note, in the list and loop pattern focus should be sent to the `h3`, which in your case would be "Afghanistan," which maybe you could update to something like "Dates served in Afghanistan." 

What do you think of those options?

**Comment by mengA6** [comment]
Thanks a lot! looks as though option 2 would be the most sound given the complexity of the IA we're dealing with. I'll go with that.

**Comment by mengA6** [comment]
Also the thing that brought this on was that our FE engineer was finding that focus went to the h2: "step 2 of 5: conditions" based on some defaults in the design system that she wasn't sure about overriding. If the intent is for the multi-page list and loop pattern to focus on the h3, should that be documented at https://design.va.gov/patterns/ask-users-for/multiple-responses ?
